core/src/main/java/org/bitcoinj/wallet/DeterministicKeyChain.java

```diff
@@ -322,17 +322,9 @@ public DeterministicKeyChain(DeterministicSeed seed, ImmutableList<ChildNumber>
      * balances and generally follow along, but spending is not possible with such a chain.
      */
     public DeterministicKeyChain(DeterministicKey watchingKey) {
-        this(watchingKey, ACCOUNT_ZERO_PATH);
-    }
-
-    /**
-     * Creates a deterministic key chain that watches the given (public only) root key. You can use this to calculate
-     * balances and generally follow along, but spending is not possible with such a chain.
-     */
-    public DeterministicKeyChain(DeterministicKey watchingKey, ImmutableList<ChildNumber> accountPath) {
```

Comment:
In many places, there were two methods.  One with the extra accountPath parameter and one without (that would then supply a default path of 0H).  In those cases, one of the methods was removed.